src/Ledger/Conway/Conformance/Certs.agda

```diff
@@ -202,7 +202,7 @@ data _⊢_⇀⦇_,CERTBASE⦈_ : CertEnv → CertState → ⊤ → CertState →
     ∙ filterˢ isKeyHash wdrlCreds ⊆ dom voteDelegs
     ∙ mapˢ (map₁ stake) (wdrls ˢ) ⊆ rewards ˢ
       ────────────────────────────────
-      ⟦ e , pp , vs , wdrls , cc ⟧ᶜ ⊢
+      ⟦ e , pp , vs , wdrls , cc ⟧ ⊢
       ⟦ ⟦ voteDelegs , stakeDelegs , rewards , ddep ⟧ᵈ
       , stᵖ
       , ⟦ dReps , ccHotKeys , gdep ⟧ᵛ
```

Comment:
```suggestion
      ⟦ ⟦ voteDelegs , stakeDelegs , rewards , ddep ⟧
      , stᵖ
      , ⟦ dReps , ccHotKeys , gdep ⟧
```
and the super `d` on line 211, super `v` on line 213, and super `cs` on lines 209 and 214.